packages/core/components/FileDetails/FileAnnotationList.tsx

```diff
@@ -94,13 +93,19 @@ export default function FileAnnotationList(props: FileAnnotationListProps) {
                 annotationValue = fileDetails.cloudPath;
             }
 
+            if (annotationValue === Annotation.MISSING_VALUE) {
+                // Nothing to show for this annotation -- skip
+                return accum;
+            }
+
             return [
                 ...accum,
                 <FileAnnotationRow
                     key={annotation.displayName}
                     className={styles.row}
                     name={annotation.displayName}
                     value={annotationValue}
+                    fmsStateIndicator={fmsStateIndicator}
```

Comment:
If this is a boolean, it should be possible to pass as just `fmsStateIndicator` without `={fmsStateIndicator}`
It also might be nice to generalize this in case we plan to re-use this styling, e.g. `shouldItalicize`

Reply:
> If this is a boolean, it should be possible to pass as just fmsStateIndicator without ={fmsStateIndicator}
The following are equivalent:
```jsx
<FileAnnotationRow fmsStateIndicator />
<FileAnnotationRow fmsStateIndicator={true} />
```
Since `fmsStateIndicator` is sometimes (usually) false, this suggestion wouldn't work without additional changes.

> It also might be nice to generalize this in case we plan to re-use this styling, e.g. shouldItalicize

I was trying to avoid a name like `shouldItalicize` that focuses on the presentation so that if we decide it should be bold or green or something instead, we don't need to also rename the variable. That said, I don't have any strong preference here.